tests/integration/kubernetes/gha-run.sh

```diff
@@ -8,6 +8,9 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+DEBUG="${DEBUG:-}"
+[ -n "$DEBUG" ] && set -x
```

Comment:
I was about to suggest the use of true/false instead of empty/not empty value, but I grep'ed the other tests scripts and most of them are using this same approach.

Reply:
I agree - but I decided to keep DEBUG consistent with most of the other scripts.